modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -143,6 +145,31 @@ spec:
       protocol: TCP
     - port: 443
 ----
+
+[id="configuringNodeSelector-example_{context}"]
+== Example nodeSelector for {kind}
+
+As a cluster administrator, you can allow or deny egress traffic to nodes in your cluster by specifying a label using `nodeSelector`. Labels can be applied to one or more nodes. The following is an example with the `type=user-node, region=east`:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: default
+spec:
+  egress:
+  - type: Allow
+    to:
+    nodeSelector:
+      region: east
```

Comment:
This is not correct adding label under "nodeSelector: " directly.  It is like this 
```
spec:
    egress:
    - to:
        nodeSelector:
          matchLabels:
            region: east
            user: qe
      type: Allow
```